Packs/FeedJSON/Integrations/FeedJSON/FeedJSON.py

```diff
@@ -12,9 +12,14 @@ def main():
             'url': params.get('url'),
             'extractor': params.get('extractor'),
             'indicator': params.get('indicator', 'indicator'),
+            'rawjson_include_indicator_type': params.get('rawjson_include_indicator_type'),
         }
     }
 
+    if params.get('auto_detect_type') and params.get('indicator_type'):
+        return_error('Indicator Type should not be set if Auto Detect Indicator Type is checked.'
+                     ' Either use Auto Detect or set manually the Indicator Type.')
+
```

Comment:
We can make it more general by adding it to: https://github.com/demisto/content/blob/master/Packs/ApiModules/Scripts/JSONFeedApiModule/JSONFeedApiModule.py#L252

WDYT?

Reply:
I didn't want to introduce a breaking change across all feeds that use this module. I would rather leave it in json feed. Other feeds usually set this anyway in a hard coded way.

Reply:
anyway also doing a commit to only do this validation on test-module command.